.github/workflows/update.yml

```diff
@@ -0,0 +1,60 @@
+name: Check update and push to AUR
+
+on:
+  schedule:
+    - cron: '37 17 * * *'
+  repository_dispatch:
```

Comment:
Recently github released workflow_dispatch which is similar to repository_dispatch but have a nice button on the webpage to manally trigger the workflow.

We can change this to workflow_dispatch. So we don't have to wait for the cron job to test the workflow.